IMAGES.rst

```diff
@@ -0,0 +1,304 @@
+      before they are reinstalled from local sources. This allows for incremental faster builds when
+      requirements change
+* ARG AIRFLOW_REPO=apache/airflow - the repository from which PIP dependencies are installed (CI optimised)
+* ARG AIRFLOW_BRANCH=master - the branch from which PIP dependencies are installed (CI optimised)
+* ARG AIRFLOW_CI_BUILD_EPOCH="1" - increasing this value will reinstall PIP dependencies from the repository
+      from scratch
+* ARG AIRFLOW_EXTRAS="all" - extras to install
+* ARG ADDITIONAL_PYTHON_DEPS="" - additional python dependencies to install
+
+Here are some examples of how CI images can built manually. CI is always built from local sources.
+
+This builds the CI image in version 3.7 with default extras ("all").
+
+.. code-block::
+
+  docker build . -f Dockerfile.ci --build-arg PYTHON_BASE_IMAGE="python:3.7-slim-buster" \
+    --build-arg PYTHON_MAJOR_MINOR_VERSION=3.7
+
+
+This builds the CI image in version 3.7 with default extras ("all").
+
+.. code-block::
+
+  docker build . -f Dockerfile.ci --build-arg PYTHON_BASE_IMAGE="python:3.7-slim-buster" \
+    --build-arg PYTHON_MAJOR_MINOR_VERSION=3.7
+
+This builds the CI image in version 3.7 with just  extras ("all").
+
+.. code-block::
+
+  docker build . -f Dockerfile.ci --build-arg PYTHON_BASE_IMAGE="python:3.7-slim-buster" \
+    --build-arg PYTHON_MAJOR_MINOR_VERSION=3.7
+
+
+Production images
+.................
+
+The following arguments can be used for CI images:
+
+* ARG PYTHON_BASE_IMAGE="python:3.6-slim-buster" - Base python image
+* ARG PYTHON_MAJOR_MINOR_VERSION="3.6" - major/minor version of Python (should match base image)
+* ARG AIRFLOW_VERSION="2.0.0.dev0" - version of Airflow
+* ARG AIRFLOW_ORG="apache" - Github organisation from which Airflow is installed (when installed from repo)
+* ARG AIRFLOW_REPO="airflow" - Github repository from which Airflow is installed (when installed from repo)
+* ARG AIRFLOW_GIT_REFERENCE="master" - reference (branch or tag) from Github repository from which
+    Airflow is installed (when installed from repo)
+* ARG REQUIREMENTS_GIT_REFERENCE="master" - reference (branch or tag) from Github repository from which
+    requirements are downloaded for constraints (when installed from repo).
+* ARG WWW_FOLDER="www" - folder where www pages are generated - it should be set to www_rbac in case
+    of 1.10 image builds.
+* ARG AIRFLOW_EXTRAS=(see Dockerfile)  Default extras with which airflow is installed
+* ARG AIRFLOW_HOME=/opt/airflow - Airflow's HOME (that's where logs and sqlite databases are stored)
+* ARG AIRFLOW_UID="50000" - Airflow user UID
+* ARG AIRFLOW_GID="50000" - Airflow group GID
+* ARG PIP_VERSION="19.0.2" - version of PIP to use
+* ARG CASS_DRIVER_BUILD_CONCURRENCY="8" - Number of processors to use for cassandra PIP install (speeds up
+       installing in case cassandra extra is used).
```

Comment:
![image](https://user-images.githubusercontent.com/8811558/78026361-6dcf3f80-7353-11ea-82b2-ace7c5795970.png)


Same here

Reply:
```
Argument | Example value | Description
-- | -- | --
``PYTHON_BASE_IMAGE`` | ``python:3.6-slim-buster`` | Base python image
``PYTHON_MAJOR_MINOR_VERSION`` | ``3.6`` | major/minor version of Python (should match base image)
``AIRFLOW_VERSION`` | ``2.0.0.dev0`` | version of Airflow
``AIRFLOW_ORG`` | ``apache`` | Github organisation from which Airflow is installed (when installed from repo)
``AIRFLOW_REPO`` | ``airflow`` | Github repository from which Airflow is installed (when installed from repo)
``AIRFLOW_GIT_REFERENCE`` | ``master`` | reference (branch or tag) from Github repository from which Airflow is installed (when installed from repo)
``REQUIREMENTS_GIT_REFERENCE`` | ``master`` | reference (branch or tag) from Github repository from which requirements are downloaded for constraints (when installed from repo).
``WWW_FOLDER`` | ``www`` - folder where www pages are generated | it should be set to www_rbac in case of 1.10 image builds.
``AIRFLOW_EXTRAS`` | (see Dockerfile) | Default extras with which airflow is installed
``AIRFLOW_HOME`` | ``/opt/airflow`` | Airflow's HOME (that's where logs and sqlite databases are stored)
``AIRFLOW_UID`` | ``50000`` | Airflow user UID
``AIRFLOW_GID`` | ``50000`` | Airflow group GID
``PIP_VERSION`` | ``19.0.2`` | version of PIP to use
``CASS_DRIVER_BUILD_CONCURRENCY`` | ``8`` | Number of processors to use for cassandra PIP install (speeds up installing in case cassandra extra is used).
```

Reply:
```
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| Argument                          | Example value                                  | Description                                                                                                                         |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| ``PYTHON_BASE_IMAGE``             | ``python:3.6-slim-buster``                     | Base python image                                                                                                                   |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| ``PYTHON_MAJOR_MINOR_VERSION``    | ``3.6``                                        | major/minor version of Python (should match base image)                                                                             |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| ``AIRFLOW_VERSION``               | ``2.0.0.dev0``                                 | version of Airflow                                                                                                                  |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| ``AIRFLOW_ORG``                   | ``apache``                                     | Github organisation from which Airflow is installed (when installed from repo)                                                      |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| ``AIRFLOW_REPO``                  | ``airflow``                                    | Github repository from which Airflow is installed (when installed from repo)                                                        |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| ``AIRFLOW_GIT_REFERENCE``         | ``master``                                     | reference (branch or tag) from Github repository from which Airflow is installed (when installed from repo)                         |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| ``REQUIREMENTS_GIT_REFERENCE``    | ``master``                                     | reference (branch or tag) from Github repository from which requirements are downloaded for constraints (when installed from repo). |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| ``WWW_FOLDER``                    | ``www`` - folder where www pages are generated | it should be set to www_rbac in case of 1.10 image builds.                                                                          |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| ``AIRFLOW_EXTRAS``                | (see Dockerfile)                               | Default extras with which airflow is installed                                                                                      |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| ``AIRFLOW_HOME``                  | ``/opt/airflow``                               | Airflow's HOME (that's where logs and sqlite databases are stored)                                                                  |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| ``AIRFLOW_UID``                   | ``50000``                                      | Airflow user UID                                                                                                                    |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| ``AIRFLOW_GID``                   | ``50000``                                      | Airflow group GID                                                                                                                   |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| ``PIP_VERSION``                   | ``19.0.2``                                     | version of PIP to use                                                                                                               |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
| ``CASS_DRIVER_BUILD_CONCURRENCY`` | ``8``                                          | Number of processors to use for cassandra PIP install (speeds up installing in case cassandra extra is used).                       |
+-----------------------------------+------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------+
```

Reply:
Thanks!